packages/node-opentelemetry/src/otel.ts

```diff
@@ -13,27 +13,34 @@ import HyperDXSpanProcessor from './spanProcessor';
 import hdx, { LOG_PREFIX as _LOG_PREFIX } from './debug';
 import { getHyperDXHTTPInstrumentationConfig } from './instrumentations/http';
 import {
+  DEFAULT_HDX_NODE_ADVANCED_NETWORK_CAPTURE,
+  DEFAULT_HDX_NODE_BETA_MODE,
+  DEFAULT_HDX_NODE_CONSOLE_CAPTURE,
+  DEFAULT_HDX_NODE_EXPERIMENTAL_EXCEPTION_CAPTURE,
+  DEFAULT_HDX_NODE_STOP_ON_TERMINATION_SIGNALS,
   DEFAULT_OTEL_EXPORTER_OTLP_TRACES_TIMEOUT,
   DEFAULT_OTEL_LOGS_EXPORTER_URL,
+  DEFAULT_OTEL_LOG_LEVEL,
   DEFAULT_OTEL_TRACES_EXPORTER_URL,
   DEFAULT_OTEL_TRACES_SAMPLER,
   DEFAULT_OTEL_TRACES_SAMPLER_ARG,
   DEFAULT_SERVICE_NAME,
-  DEFAULT_OTEL_LOG_LEVEL,
 } from './constants';
 import { hyperDXGlobalContext } from './context';
+import { initSDK as initSentrySDK } from './sentry/node';
```

Comment:
this feels confusing, why don't we just pick a consistent name everywhere instead of renaming?

Reply:
because we have our own initSDK. I guess we can import * to make it consistent